components/toast/stories/toast.stories.js

```diff
@@ -84,6 +109,39 @@ Positive.parameters = {
 	chromatic: { disableSnapshot: true },
 };
 
+/**
+ * Toasts must include text to communicate a message. Write the text as concisely as possible while still being clear about what has happened or is happening.
+ * 
+ * Writing for toasts depends on the nature of the message, whether it’s communicating confirmation, information, or an error. For all kinds of toasts, keep the text to fewer than 2 lines in English, since it will become longer when localized. View the [toast content standards](https://spectrum.adobe.com/page/toast/#Content-standards) for writing guidelines.
+*/
+
+export const Text = Template.bind({});
+Text.tags = ["!dev"];
+Text.args = {
+	message: "All files archived",
+	inlineButtonLabel: ""
+};
+
+/**
+ * When the message text is too long for the horizontal space available, it wraps to form another line. 
+*/
+
+export const Wrapping = ToastWrapOptions.bind({});
+Wrapping.tags = ["!dev"];
+Wrapping.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * A toast can have up to one action: [a static white, secondary, outline button.](?path=/docs/components-button--docs#static-white---secondary) This label should be kept concise, and it should only be used when there’s a direct action available that is related to the toast text.
```

Comment:
```suggestion
 * A toast can have up to one action: [a static white, secondary, outline button](?path=/docs/components-button--docs#static-white---secondary). This label should be kept concise, and it should only be used when there’s a direct action available that is related to the toast text.
```

Minor nit to move the period to outside the link.